src/IssueCreator/Main.cs

```diff
@@ -162,7 +179,7 @@ private async void UpdateEpicListAsync()
 
             // add just the open issues
             cboEpics.Items.Clear();
-            cboEpics.Items.AddRange(issues.Where((issue) => issue.Issue.State == ItemState.Open).ToArray());
+            cboEpics.Items.AddRange(issues.Where((issue) => StringComparer.OrdinalIgnoreCase.Equals(issue.Issue.State, Octokit.ItemState.Open.ToString())).ToArray());
```

Comment:
Why waste time copying to an array when `AddRange` most likely takes an `IEnumerable<T>` anyway? As long as its copying items, this is just wasting time and memory with no benefit to "sealing" the enumerable.

Reply:
Actually, the Items collection on the ComboBox only takes in an object[] ...